.coveralls.yml

```diff
@@ -0,0 +1 @@
+repo_token: jbGqxIWKg9JHvWBhix0TeKxDLRDvZpzwi
```

Comment:
Well done @ocranbillions. Is it okay to have the repo token in public?

Reply:
You're right @Hadeneekeh It's safe to make it private. I could make it an env variable, but i'm lost at how to read that into a YAML. Will do some research on this. 
Anyone with an idea of how to fix this?

Reply:
Hi @ocranbillions, I think we don't need to specify the repo token since it's a public repo. Only Private repos need the token specified explicitly. 
[Check this out](https://github.com/lemurheavy/coveralls-public/issues/208)

Reply:
Chux is right

Reply:
Morning @chuxmykel will the coverage report still work without the repo token in the .coveralls.yml file?

Reply:
Is so, then I should probably just take out the yml file entirely

Reply:
Yes @ocranbillions. It is redundant.